instrumentation/ratpack-1.4/src/main/java/io/opentelemetry/javaagent/instrumentation/ratpack/TracingHandler.java

```diff
@@ -25,7 +25,7 @@
    */
   public static final AttributeKey<io.opentelemetry.context.Context> SERVER_ATTRIBUTE_KEY =
       AttributeKey.valueOf(
-          "io.opentelemetry.javaagent.instrumentation.netty.v4_1.server.HttpServerTracingHandler.context");
+          "io.opentelemetry.javaagent.instrumentation.netty.v4_1.AttributeKeys.context");
```

Comment:
Should this use class as well?

Reply:
done, this required adding a dependency on the netty instrumentation, which seems reasonable